modules/accredited_representative_portal/spec/sidekiq/accredited_representative_portal/redact_power_of_attorney_requests_job_spec.rb

```diff
@@ -9,6 +9,10 @@
 # rubocop:disable Metrics/ModuleLength
 module AccreditedRepresentativePortal
   RSpec.describe RedactPowerOfAttorneyRequestsJob, type: :job do
+    before do
+      Flipper.enable(:accredited_representative_portal_full_poa_redaction)
```

Comment:
Oh I think platform is going to ask us to change these to:
```
allow(Flipper).to receive(:enabled?).with(:accredited_representative_portal_full_poa_redaction).and_return(true)
```

Reply:
🎉 thank you!